docs/userguide/dockernetworks.md

```diff
@@ -0,0 +1,511 @@
+<!--[metadata]>
++++
+title = "Docker container networking"
+description = "How do we connect docker containers within and across hosts ?"
+keywords = ["Examples, Usage, network, docker, documentation, user guide, multihost, cluster"]
+[menu.main]
+parent = "smn_containers"
+weight = 3
++++
+<![end-metadata]-->
+
+# Docker container networking
+
+So far we've been introduced to some [basic Docker
+concepts](/userguide/usingdocker/), seen how to work with [Docker
+images](/userguide/dockerimages/) as well as learned about basic [networking
+and links between containers](/userguide/dockerlinks/). In this section
+we're going to discuss how you can take control over more advanced 
+container networking.
+
+This section makes use of `docker network` commands and outputs to explain the
+advanced networking functionality supported by Docker.
+
+# Default Networks
+
+By default, docker creates 3 networks using 3 different network drivers :
+
+```
+$ sudo docker network ls
+NETWORK ID          NAME                DRIVER
+7fca4eb8c647        bridge              bridge
+9f904ee27bf5        none                null
+cf03ee007fb4        host                host
+```
+
+`docker network inspect` gives more information about a network
+
+```
+$ sudo docker network inspect bridge
+{
+    "name": "bridge",
+    "id": "7fca4eb8c647e57e9d46c32714271e0c3f8bf8d17d346629e2820547b2d90039",
+    "driver": "bridge"
+}
+```
+
+By default containers are launched on Bridge network
+
+```
+$ sudo docker run -itd --name=container1 busybox
+f2870c98fd504370fb86e59f32cd0753b1ac9b69b7d80566ffc7192a82b3ed27
+
+$ sudo docker run -itd --name=container2 busybox
+bda12f8922785d1f160be70736f26c1e331ab8aaf8ed8d56728508f2e2fd4727
+```
```

Comment:
This code snippet doesn't demonstrate statement that precedes it. I think it'd be better to combine it with the `inspect` command below, which *does* demonstrate that the containers have joined the `bridge` network.